SOTS_2019_WDR.tex

```diff
@@ -0,0 +1,502 @@
+    \item The \secondaryfull{} is the backup for \primary{}, and communicates with \ops{}. If \primary{} becomes incapacitated, \secondary{} is responsible for removing them from danger.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end personnel required section -- }}}
+
+% begin sign off -- {{{
+\subsection{Sign-Off}
+\textit{To be completed by all test personnel after reading and familiarization with procedures}
+\begin{checklist}
+    \item \opsfull      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \controlfull      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \primaryfull  \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \secondaryfull\tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end sign off -- }}}
+\newpage{}
+
+\section{[N1] Final Setup and "Pre-Launch" Checks} \flag{Blue}
+% begin prior to start section -- {{{
+\subsection{Prior to Start}
+\begin{checklist}
+    \item Ensure that the following procedures are complete:
+    \begin{checklist}
+        \item Rocket Assembly procedure
+        \item RLCS Setup procedure
+        \item Launch Tower Setup procedure
+    \end{checklist}
+    \item Ensure that all personnel as defined above are available and have completed the sign-off.
+    \item Ensure that the following personnel have walkie-talkies and communication is functional:
+    \begin{checklist}
+        \item \ops{}
+        \item \control{}
+        \item \primary{}
+        \item \secondary{}
+    \end{checklist}
+    \item Ensure that \ops{} is in possession of the system control key.
+    \item Ensure that the client-side RLCS box is powered off.
+    \item Ensure that the locations of Launch Control, Launch Tower, and the Minimum Safe Distance are clearly defined.
+    
+\end{checklist}
+\begin{center}
+\begin{tabu} to \textwidth { |[1pt]X[c]|[1pt]X[c]|[1pt]X[c]|[1pt] }
+\tabucline[1pt]{}
+Launch Control & Launch Tower & Minimum Safe Distance \\
+\tabucline[1pt]{}
+\vspace{1in} & & \\
+\tabucline[1pt]{}
+\end{tabu}
+\end{center}
+\setcounter{checklistnum}{0}
+% end prior to start section -- }}}
+
+% begin Fill System Check section -- {{{
+\subsection{Nominal Procedure}
+\begin{checklist}
+    \item \primary{}: Confirm that the following valves are initially closed:
+        \begin{checklist}
+        \item Cylinder Valve
+        \item Remote Fill Valve
+        \item Parallel Fill Valve
+        \item Series Fill Valve
+        \item Line Vent Valve
+        \item Parallel Vent Valve
+    \end{checklist}
+    \item \primary{}: Confirm that the ignition connectors are disconnected from the rocket.
+    \item \control{}: Power on the client-side RLCS box.
+    \item \control{} and \secondary: Confirm that the following actuators fail to move:
```

Comment:
This might be slightly different behaviour because of how rocketcan works, but I'm not positive. Let's leave it like this for now, and adjust if necessary.